fdpa-resolutions.html

```diff
@@ -86,5 +28,159 @@ <h2>Fair Districts PA - Local Resolutions</h2>
         &nbsp; | &nbsp; Click a county or municipality for more info &nbsp; | &nbsp;
         <a href="https://tinytake.com/how-to-take-a-screenshot/">Help with screen shots</a></p>
     <div id="map-canvas"></div>
+    <script src="https://www.promisejs.org/polyfills/promise-7.0.4.min.js">
+    </script>
+    <script type="text/javascript">
+      var map;
+      var layer_0;
+      var layer_1;
+      function initialize() {

```

Comment:
initialize() is basically identical to the original JavaScript. It now gets called after dynamically injecting the google maps script with either the production or the development key.